doc/quick_start/contributor-guide.rst

```diff
@@ -0,0 +1,289 @@
+
+   To replace the current layer with the one you are working on, modify your <builddir>/conf/bblayers.conf file as appropriate
+   or use the Yocto Project tools ``bitbake-layers`` command instead of manually editing the ``bblayers.conf`` file::
+
+      $ bitbake-layers show-layers          # add-layer, remove-layer are other options...
+
+   See the `Yocto Project Managing Layers`_ documentation for more usage details.
+
+   When readh, run bitbake to start the build:::
+
+      $ bitbake -k ostro-image-noswupd        # for example, other target images are available too
+
+#. Commit your changes and rebase onto master
+   After you’ve tested and verified your change does what was intended, you can commit your change locally. 
+   Make sure that you follow the `Code Review Process Guidelines`_ described earlier in this document:::
+
+      $ git commit -a -s                    # follow guidelines for the commit message
+      $ git push origin my-patch-branch     # push your local branch up to your forked GitHub repo
+
+   Depending on how long you have worked on your patch, it may be that the master branch has evolved since you branched 
+   it off. If that is the case, you should rebase your working branch onto master before sending the Pull Request (PR):::
+
+
+      $ git rebase upstream/master
+
+#. Create a Pull Request (PR)
+   Once your change is in your forked version (up on GitHub), use your web browser to submit your PR:
+
+     * Navigate to your branch: https://github.com/<username>/meta-ostro/tree/my-patched-branch (the branch name you
+       created earlier).
+     * Click on "Compare & pull request" button From there you can see your changes and create a Pull Request (PR) to the 
+       master branch for that component.
+
+
+7. Respond to Pull Request (PR) Comments
+   You may be asked to update or re-work your patch as part of the review process. 
+   The easiest way to keep the discussion going in the same Pull Request is to force-push a revised commit to your 
+   forked repository. GitHub will automatically update the Pull Request with the latest changes.  Using amended
+   commits is preferred over a PR with multiple commits and helps make reviewing the cumulative changes much easier:::
+
+
+      $ git commit --amend
+      $ git push -f origin my-patched-branch      # force the push back to your forked copy on GitHub
+
+   Be sure to add a comment to your amended commit message saying what was changed and that this update was forced-pushed, 
+   otherwise reviewers will not get notification about the change.  
+   Once the reviewers and maintainers accept your changes, they will be merged and incorporated in the Ostro OS 
+   next time the maintainers run the combo-layer script. 
+
+Once your changes have been merged, you can clean your local branches and go back to using the layer that is part of 
+the Ostro project (i.e., revert the changes you have made in <builddir>/conf/bblayers.conf)
```

Comment:
Can we add a similar note as above, i.e. that we recommend people use bitbake-layers for this operation?